common/src/main/java/org/opensearch/ml/common/conversational/ActionConstants.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2023 Aryn
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.opensearch.ml.common.conversational;
+
+/**
+ * Constants for conversational actions
+ */
+public class ActionConstants {
+
+    /** name of conversation Id field in all responses */
+    public final static String CONVERSATION_ID_FIELD = "conversationId";
+
+    /** name of list of conversations in all responses */
+    public final static String RESPONSE_CONVERSATION_LIST_FIELD = "conversations";
+    /** name of list on interactions in all responses */
+    public final static String RESPONSE_INTER_LIST_FIELD = "interactions";
+    /** name of interaction Id field in all responses */
+    public final static String RESPONSE_INTER_ID_FIELD = "interactionId";
```

Comment:
Is this a index field name? If yes, we should follow snake case ,  `interaction_id`

Reply:
technically, no, it only appears in the API responses. That said, there are some index field names (`"conversationId", "numInteractions", ...`) that should go to snake_case, and for consistency, I'll convert the API fields as well